src/server/scripts/Outland/GruulsLair/boss_gruul.cpp

```diff
@@ -114,6 +114,8 @@ class boss_gruul : public CreatureScript
                 case EVENT_HURTFUL_STRIKE:
                     if (Unit* target = SelectTarget(SELECT_TARGET_TOPAGGRO, 1, 5.0f))
                         me->CastSpell(target, SPELL_HURTFUL_STRIKE, false);
+                    else if (me->GetVictim())                                           // If no target was found for Hurtful Strike, target the MT
+                        me->CastSpell(me->GetVictim(), SPELL_HURTFUL_STRIKE, false);
```

Comment:
wouldn't the top aggro target be the tank if there are no other targets?

Reply:
Im not good in telling what the code does, but in the current original master script, Gruul does not attack the MT with Hurtful Strike.

Reply:
Interesting, well you could use just `else` instead of `else if` since there are no other checks below

Reply:
Im kinda cautious cause i dont really forsee all effects of my changes.
Also, could a plain "else" possibly bug out if Gruul has no target for some reason?

Reply:
Well a simple else would be anything that does not meet the requirement of the if.

But if he has no target he cant cast it since it would not have a victim to pass `me->GetVictim()` if I'm not misstaken